_sass/_docs.scss

```diff
@@ -5,5 +5,9 @@
     will-change: transform;
     padding-top: 4em;
   }
+  
+  .content h2 {
+    text-transform: none;
+  }
```

Comment:
The block of styles from FlatDoc that sets `text-transform: uppercase;` contains two other rules:

~~~css
.content h1, .content h2 {
    text-transform: uppercase;
    letter-spacing: 1px;
    font-size: 1.5em;
}
~~~

Of these, the `letter-spacing: 1px;` rule was probably added only because of the `text-transform` rule. After all, [letter spacing](https://practicaltypography.com/letterspacing.html) is usually added to runs of capital letters. Thus, I suggest disabling that rule too:

~~~suggestion
    letter-spacing: initial;
  }
~~~